### PR TITLE
Fixed bugs

### PR DESCRIPTION
**Pull request by SaranchukVadym**

### CODE REVIEW

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/CandidateEventService.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import com.exadel.team2.sandbox.web.candidate_event.CreateCandidateEventDto;
 import com.exadel.team2.sandbox.web.candidate_event.ResponseCandidateEventDto;
+import cz.jirutka.rsql.parser.RSQLParserException;
```
Comment (akarzhaneuski) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/CandidateEventService.java:5]: Unused import

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
+  - changeSet:
+      id: run-fix-table-candidate
+      author: Saranchuk Vadim
+      preConditions:
```
Comment (akarzhaneuski) [inline sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml:5]: Remove empty preconditions

**Review by akarzhaneuski — CHANGES_REQUESTED** [review]

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/CandidateEventService.java (hunk 1 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/CandidateEventService.java:5]: corrected

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml (hunk 2 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml:5]: corrected

**Pull request closed without merging**